docker/symfony_app/Dockerfile

```diff
@@ -11,6 +11,9 @@ RUN apk add --no-cache zip libzip-dev
 RUN docker-php-ext-configure zip --with-libzip
 RUN docker-php-ext-install zip
 
+# we need mysqli extension for phpmyadmin
+RUN docker-php-ext-install mysqli && docker-php-ext-enable mysqli
```

Comment:
Hello @korzol I think that we do not need to have any changes in symfony_app container for this task.

Reply:
Hello @ceckoslab 

Yeah, you are right. Already removed